scrapybot/scrapybot/spiders/btcphilosophy.py

```diff
@@ -19,10 +21,18 @@ class BtcphilosophySpider(CrawlSpider):
 
     def parse_item(self, response):
         item = {}
-        article = response.xpath("//article").get()
+        soup = BeautifulSoup(response.text, "html.parser")
+        script_tags = soup.find_all("script")
+        res = script_tags[-1]
+        json_object = json.loads(res.contents[0])
+        payload = json_object["payload"]
+        article = payload["blob"]["richText"]
         item["id"] = "btcphilosophy-" + str(uuid.uuid4())
-        item["title"] = "[Bitcoin Dev Philosopy] " + response.xpath("//article/div/h2/text()").get()
 
+        item["title"] = (
+            "[Bitcoin Dev Philosopy] "
+            + BeautifulSoup(article, "html.parser").find("h2").text
```

Comment:
I'm getting errors for some of the files where it can't parse the `h2` header. If extracting `h2` header here isn't completely necessary, we can have a priority of headings like:
```suggestion
            + BeautifulSoup(article, "html.parser").find(['h1', "h2", 'h3', 'h4', 'h5', 'h6']).text
```